src/lib/PlanningForm.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+  import { gjScheme, deleteIntervention, formOpen } from "../stores";
+  import type { InterventionProps, PlanningProps } from "../types";
+
+  export let id: number;
+  export let allProperties: InterventionProps;
+
+  // TODO Should we start with these defaults, or make everything optional?
+  allProperties.planning ||= {
+    name: "",
+    notes: "",
+    reference_type: "preapp",
+    size: 0,
+    size_units: "number of units",
+    triage: "No Comment",
+    recommendation: "No Comment",
+  };
+  let props: PlanningProps = allProperties.planning!;
+
+  const referenceTypes = [
+    "preapp",
+    "outline",
+    "reserved matters",
+    "local plan",
+  ];
+  const sizeUnits = [
+    "number of units",
+    "floorspace (Gross Floor Area)",
+    "area (Hectare)",
+  ];
+  const triages = ["No Comment", "Standing Advice", "Toolkit Assessment"];
```

Comment:
There's repetition here with `types.ts`. It's not onerous to keep in sync right now, and TS should help us with accidental typos or mismatches. But maybe there's a way to do TS build-time reflection and get all string cases of an enum?